packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -446,12 +455,13 @@ const stepFlow = async (
 
   const provideAgoricNoble = () =>
     Promise.all([
-      provideCosmosAccount(orch, 'agoric', kit),
-      provideCosmosAccount(orch, 'noble', kit),
+      provideCosmosAccount(orch, 'agoric', kit, portfolioTrace),
+      provideCosmosAccount(orch, 'noble', kit, portfolioTrace),
     ]);
 
-  for (const move of moves) {
-    trace('wayFromSrcToDesc?', move);
+  for (const [i, move] of entries(moves)) {
+    const indexTrace = portfolioTrace.sub(`move${i}`);
```

Comment:
seems like it should be...

```suggestion
    const moveTrace = portfolioTrace.sub(`move${i}`);
```

Reply:
In light of further suggestions, changed to `traceMove`. Done